core/test/integration/model/model_posts_spec.js

```diff
@@ -1681,12 +1681,12 @@ describe('Post Model', function () {
                 found.length.should.equal(50);
                 return PostModel.destroyByAuthor(authorData);
             }).then(function (results) {
-                // User 1 has 13 posts in the database
-                results.length.should.equal(13);
+                // User 1 has 10 posts in the database
+                results.length.should.equal(10);
                 return PostModel.findAll({context: {internal: true}});
             }).then(function (found) {
-                // Only 37 should remain
-                found.length.should.equal(37);
+                // Only 40 should remain
+                found.length.should.equal(40);
```

Comment:
can you explain the change here?
the only thing i saw was that we add one more contributor user to the "extra users" fn in our test utils. maybe i am overseeing that.